docs/README.md

```diff
@@ -51,9 +51,7 @@ systems and pass the benefits along to their customers.
 
 ### Contributors
 
-With a clear project scope, lightweight governance model, and clean lines of
-separation between pluggable components, the Knative project establishes an
-efficient contributor workflow.
+While not a part of a software foundation, Knative provides contributors with a clear project scope and an open governance model where multiple companies are represented. The Knative project strives for both clean lines of separation between pluggable components and an efficient contributor workflow.
```

Comment:
Not LGTM
This is just asking for public disagreement and it would suggest that this is best solved by the SC, so please revert the “open governance” change